docs/checkers/index.md

```diff
@@ -2557,6 +2557,16 @@ and `pdb.set_trace()`) are found. These breakpoints should be removed in product
 
 ```
 
+(E4703)=
+
+## Modified iterating set
+
+This error occurs when a set is modified inside a for loop by adding or removing items from the `set`. Other types of modification are actually okay, and do not trigger the error. A copy of the `set` can be used instead.
+
+```{literalinclude} /../examples/pylint/e4702_modified_iterating_set.py
```

Comment:
This isn't the correct path. You can (and should) check your work by building the documentation pages on your own computer: https://github.com/pyta-uoft/pyta#generating-the-docs.